x-pack/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -135,6 +145,24 @@ export class HeadlessChromiumDriver {
     });
   }
 
+  private async launchDebugger() {
+    // HACKY: Puppeteer doesn't expose a top-level API to set debug statements and get
+    // identifiers like a page's ID. These are required in order to have an effective debugging experience.
```

Comment:
Sorry, but what is meant by `page's ID`?

Reply:
Also, hopefully this is not a Puppeteer 2.0 issue?

Reply:
I'll update this comment for clarity sake